eclair-core/src/main/scala/fr/acinq/eclair/payment/Relayer.scala

```diff
@@ -333,4 +329,10 @@ object Relayer {
       case _ => requestedShortChannelId // we don't have a channel_update for this short_channel_id
     }
   }
+
+  def updateChannelUpdatesIfPresent(channelUpdates: Map[ShortChannelId, OutgoingChannel], shortChannelId: ShortChannelId, commitments: Commitments): Map[ShortChannelId, OutgoingChannel] =
```

Comment:
If this is used from several different packages then it probably doesn't belong here. There must be a better way to factor this.